### DISCUSSION TITLE
How to know if a connection is actually closed?

### DISCUSSION THREAD

**Discussion by glwbr** [qruto/laravel-wave#68] [Q&A]
I'm using Wave for handling PaymentNotifications (OrderConfirmations). When an order is updated, I send an Event to my frontend (Vue.js), triggering some actions. However, in production, we observed an increase in server resource consumption after deploying this strategy. Is there a way to ensure that a connection is properly closed? Currently, I'm testing a setTimeout to close the connection on the frontend, but I'm unsure if this is effective.
![Code-Example](https://github.com/qruto/laravel-wave/assets/6403699/fa0aaa1b-c8ad-48f6-8562-953e67dd7129)

**Comment by slavarazum** [comment]
To drop stale connections, Wave uses a ping event. By default, it sends a ping event for every new SSE connection if the last event was earlier than 30 seconds. Check [PingConnections](https://github.com/qruto/laravel-wave/blob/main/src/Http/Middleware/PingConnections.php) middleware.
If you need to close slate connections more frequently, you can [manual control the ping event](https://github.com/qruto/laravel-wave?tab=readme-ov-file#option-2-manual-ping-control).
